public/app/features/alerting/unified/components/rule-editor/FolderAndGroup.tsx

```diff
@@ -213,8 +224,7 @@ export function FolderAndGroup({ groupfoldersForGrafana }: { groupfoldersForGraf
                     getOptionLabel={(option: SelectableValue<string>) => (
                       <div>
                         <span>{option.label}</span>
-                        {/* making the assumption here that it's provisioned when it's disabled, should probably change this */}
```

Comment:
we remove this assumption using this new field `isProvisioned`

Reply:
Nice!